package-lock.json

```diff
@@ -0,0 +1,4820 @@
+{
```

Comment:
IOTA repositories intentionally doesn't include package-lock.json. We can open a discussion thread about the convenience of having pacakges-lock.json included in the repository or not :) but by the moment I'll suggest to leave it out this PR, please.

Reply:
Package lock removed. Fixed 6f5a131